scripts/sync.ts

```diff
@@ -1,19 +1,48 @@
 import fs from "fs";
 
+import {FetchingJSONSchemaStore, InputData, JSONSchemaInput, quicktype} from "quicktype-core";
+
+const ECOSYSTEM_DATA_URL = "https://thunderstore.io/api/experimental/schema/dev/latest/";
+const ECOSYSTEM_JSON_SCHEMA_URL = "https://thunderstore.io/api/experimental/schema/ecosystem-json-schema/latest/";
+const ECOSYSTEM_DATA_PATH = "./src/assets/data/ecosystem.json";
+const ECOSYSTEM_JSON_SCHEMA_PATH = "./src/assets/data/ecosystemJsonSchema.json";
+const ECOSYSTEM_DATA_TYPES_PATH = "./src/assets/data/ecosystem.d.ts";
```

Comment:
I'm considering changing this from ecosystem.d.ts` to something like `ecosystemTypes.ts` so Enums can be imported instead of having to be redefined in the mod manager.

Reply:
This is now done in https://github.com/ebkr/r2modmanPlus/pull/1751